boxoffice/assets/js/views/org.js

```diff
@@ -85,36 +73,36 @@ export const OrgView = {
           orgName: org_name,
           orgTitle: org_title,
           item_collections: item_collections,
-          icForm: '',
           showAddForm: DEFAULT.hideForm,
-          editIcForm: ''
         },
-        components: {ICForm: ICForm, EditICForm: EditICForm},
-        showNewIcForm: function (event) {
-          this.set('showAddForm', DEFAULT.showForm);
+        components: {ICForm: ICForm},
+        showIcForm: function (event, action) {
+          if (action === "edit") {
+            let ic = event.keypath,
```

Comment:
Can you help explain what's happening here, I'm a little confused.

Reply:
In case the action is `edit` Item collection, client does an ajax call to fetch the item collection edit form.